libgstc/python/pygstc/gstc.py

```diff
@@ -829,6 +831,26 @@ def pipeline_create(self, pipe_name, pipe_desc):
         parameters = self._check_parameters([pipe_name, pipe_desc], [str, str])
         self._send_cmd_line(['pipeline_create'] + parameters)
 
+    def pipeline_create_ref(self, pipe_name, pipe_desc):
+        """
+        Create a new pipeline based on the name and description using refcount.
+        The refcount works similarly to GObject references. If the command
+        is called but the refcount is greater than 0 nothing will happen
+        and the refcount will increment.
+
+        Parameters
+        ----------
+        pipe_name: string
+            The name of the pipeline
+        pipe_desc: string
+            Pipeline description (same as gst-launch-1.0)
+        """
+
+        self._logger.info('Creating pipeline by reference %s with description "%s"'
```

Comment:
New style string formatting is preferred:
```
'Creating pipeline by reference {} with description {}'.format(pipe_name, pipe_desc)
```
instead of 
```
'Creating pipeline by reference %s with description "%s"' % (pipe_name, pipe_desc)
```
See:
https://realpython.com/python-string-formatting/

Reply:
Nice!